HW03/Alexey_Zalyotov/lib/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :name, :surname
+  attr_accessor :notifications
+
+  def initialize(name:, surname:)
+    @name = name
+    @surname = surname
+    @notifications = []
+  end
+
+  def show_notifications
+    puts "____________________________\n" \
+        "#{self.class}`s notifications:"
+    notifications.empty? ? (puts 'No notifications') : (puts notifications)
+    puts
+  end
+
+  def read_notifications!
+    @notifications.clear
```

Comment:
I think it should work 
```suggestion
    notifications.clear
```

Reply:
Done